File: programmers/Lv.0/숨어있는 숫자의 덧셈 (1).js
// 문자열 my_string이 매개변수로 주어집니다. my_string안의 모든 자연수들의 합을 return하도록 solution 함수를 완성해주세요.
function solution(my_string) {
  let sum = 0;
  for (let value of my_string) {
    if (Number(value)) {
      sum += Number(value);
    }
  }
  return sum;
}

File: programmers/Java/Lv.0/숨어있는 숫자의 덧셈 (1).java
// 문자열 my_string이 매개변수로 주어집니다. my_string안의 모든 자연수들의 합을 return하도록 solution 함수를 완성해주세요.
class Solution {
    public int solution(String my_string) {
        String[] arr = my_string.split("");
        int answer = 0;
        for(int i = 0; i < arr.length; i++){
            char c = my_string.charAt(i);
            if(Character.isDigit(c)){
                answer += Integer.parseInt(arr[i]);
            }
        }
        return answer;
    }
}